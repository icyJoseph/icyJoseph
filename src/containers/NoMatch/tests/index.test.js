import React from "react";
import { shallow } from "enzyme";
import { NoMatch } from "../";

describe("404 Page", () => {
  const wrapper = shallow(<NoMatch />);
  it("renders", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
