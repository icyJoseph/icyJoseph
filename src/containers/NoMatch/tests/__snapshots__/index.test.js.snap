// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 Page renders 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "404",
    },
    "ref": null,
    "rendered": "404",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "404",
      },
      "ref": null,
      "rendered": "404",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
